src/src_V2/components/container/Container.jsx

```diff
@@ -2,51 +2,52 @@ import { useState } from 'react';
 import { Box } from '@mui/material';
 import { styled } from '@mui/material/styles';
 
-import Header from './header/Header';
-import Sidebar from './Sidebar';
+import Header from '../header/Header';
+import Sidebar from '../sidebar/Sidebar';
 
-import { CONTAINER_SIZES } from '../../utils';
-
-const { DRAWER_EXPANDED_WIDTH, DRAWER_COLLAPSED_WIDTH } = CONTAINER_SIZES;
+import Layout from '../layout/Layout';
 
 const Main = styled('main')(({ theme, open }) => ({
   flexGrow: 1,
   transition: theme.transitions.create('margin', {
     easing: theme.transitions.easing.sharp,
     duration: theme.transitions.duration.leavingScreen,
   }),
-  marginLeft: DRAWER_COLLAPSED_WIDTH,
   ...(open && {
     transition: theme.transitions.create('margin', {
       easing: theme.transitions.easing.easeOut,
       duration: theme.transitions.duration.enteringScreen,
     }),
-    marginLeft: DRAWER_EXPANDED_WIDTH,
   }),
 }));
 
 const boxStyles = {
-  width: '100%',
   position: 'relative',
-  display: 'inline-block',
+  display: 'flex',
+  minHeight: '91.4vh',
+  marginTop: '2px',
 };
 
 /**
  * @description - Container component which includes the following components in it, Sidebar, Appbar and Main component
  *                with some ui styles to support the Sidebars behavior(expand/collapse);
  * @param children - The children elements.
  * @param managerSidebarConfig - The configs for sidebar menu options.
+ * @param showLayout - Whether to show the Layout component
+ * @param title - The Title of Layout if there is one
  * @returns {JSX.Element}
  */
-export default function Enhancer({ children, managerSidebarConfig }) {
-  const [openSidebar, setOpenSidebar] = useState(false);
+export default function Enhancer({ children, managerSidebarConfig, showLayout, title }) {
+  const [openSidebar, setOpenSidebar] = useState(true);
 
   return (
     <Box>
       <Header />
-      <Box style={boxStyles}>
+      <Box sx={boxStyles}>
         <Sidebar pageConfigs={managerSidebarConfig} open={openSidebar} setOpen={setOpenSidebar} />
-        <Main open={openSidebar}>{children}</Main>
+        <Main open={openSidebar}>
+          {showLayout ? <Layout title={title}>{children}</Layout> : children}
+        </Main>
       </Box>
     </Box>
   );
```

Comment:
I think we can use the Layaout Comnponent where we need it and send as a children to Container component and also we can remove conditioanl rendering for Layout component and lead to use it whenever we need it. How do you think?

Reply:
Agreed. Will change accordingly. Thanks